crates/matrix-sdk-ui/src/timeline/inner.rs

```diff
@@ -958,3 +1215,88 @@ async fn fetch_replied_to_event(
     };
     Ok(res)
 }
+
+fn update_timeline_reaction(
+    mut state: MutexGuard<'_, TimelineInnerState>,
```

Comment:
```suggestion
    state: &mut TimelineInnerState,
```

might require an extra `&mut` in front of the argument for calls, but makes the signature easier to read.

Reply:
[`2092493`](https://github.com/matrix-org/matrix-rust-sdk/pull/2118/commits/20924938dd0d108eca25c6b2e4a5fc2dde29902d)